src/include/samba-server/dialogs.rb

```diff
@@ -140,7 +142,7 @@ def subvolume?(path)
     def snapper_cfg?(path)
       return false unless path
 
-      if 0 == SCR.Execute(path(".target.bash"), "grep 'SUBVOLUME=\"#{path}\"' /etc/snapper/configs/*")
+      if 0 == SCR.Execute(path(".target.bash"), "/usr/bin/grep SUBVOLUME=\\\"#{path.shellescape}\\\" /etc/snapper/configs/*")
```

Comment:
This backslash and double quote escaping is tricky and fragile, I'd leave that for `#shellescape` as well.

```ruby
pattern = "SUBVOLUME=\"#{path}\""
"/usr/bin/grep #{pattern.shellescape} /etc/..."
```

This is a bit more readable and avoid typos.

Reply:
yep, I also consider it, so lets use it.